impl/integration/env.go

```diff
@@ -130,14 +130,19 @@ func NewEnv() (*Env, error) {
 	pb.RegisterKeyTransparencyServer(gsvr, server)
 
 	// Sequencer
-	seq := sequencer.New(logEnv.Log, mapEnv.Map, entry.New(), domainStorage, mutations, queue)
+	seq := sequencer.New(logEnv.Log, logEnv.Admin, mapEnv.Map, entry.New(), domainStorage, mutations, queue)
 	// Only sequence when explicitly asked with receiver.Flush()
 	d := &domaindef.Domain{
-		DomainID: domainPB.DomainId,
-		LogID:    domainPB.Log.TreeId,
-		MapID:    domainPB.Map.TreeId,
+		DomainID:    domainPB.DomainId,
+		LogID:       domainPB.Log.TreeId,
+		MapID:       domainPB.Map.TreeId,
+		MinInterval: 60 * time.Hour,
```

Comment:
60 hours?

Reply:
This supports only sequencing when `receiver.Flush` is manually called.